cmd/kubeadm/app/cmd/phases/certs.go

```diff
@@ -60,8 +60,8 @@ var (
 	apiServerCertLongDesc = fmt.Sprintf(normalizer.LongDesc(`
 		Generates the API server serving certificate and key and saves them into %s and %s files.
 		
-		The certificate includes default subject alternative names and additional sans eventually provided by the user;
-		default sans are: <node-name>, <apiserver-advertise-address>, kubernetes, kubernetes.default, kubernetes.default.svc, 
+		The certificate includes default subject alternative names and additional SANs provided by the user;
```

Comment:
Maybe capitalize `Subject Alternative Names` so that it's more clear that `SAN` (used later) is abbreviating that (applies below too).